pkg/aspect/inventory.go

```diff
@@ -18,18 +18,22 @@ import (
 	"istio.io/mixer/pkg/config"
 )
 
-// ManagerFinder find an aspect Manager given aspect kind.
-type ManagerFinder interface {
-	// ByImpl queries the registry by adapter name.
-	FindManager(kind string) (Manager, bool)
-}
-
 // APIBinding associates an aspect with an API method
 type APIBinding struct {
 	Aspect Manager
 	Method config.APIMethod
 }
 
+// Name of all supported aspect kinds.
+const (
+	AccessLogKind = "access-logs"
```

Comment:
I was guessing that it was a lint issue (not documenting an exported element). I _feel_ like that used to be a golint issue, but maybe not now? or maybe my memory is hazy?